lib/collection.js

```diff
@@ -159,14 +159,14 @@ const BookshelfCollection = (module.exports = CollectionBase.extend(
         this.sync(options)
           .select()
           .bind(this)
+          
+          // Now, load all of the data onto the collection as necessary.
           .tap(function(response) {
```

Comment:
it seems like bookshelf isn't using tap in the way it is intended to be used. These are behaving like .then.